README.md

```diff
@@ -87,3 +87,16 @@ user_track_ids.each do |id|
   CreateSolution.(ut.user, ut.track.exercises.core.first) if ut.solutions.size == 0
 end
 ```
+
+## Troubleshooting
+### MySQL < 5.7
+The following error is seen using MySQL prior to version 5.7 a discussed at gogs/gogs#4894.
+```
+$ bundle exec rake exercism:setup
+Mysql2::Error: Specified key was too long; max key length is 767 bytes:
+```
+If your system doesn't have a pre-built 5.7 package, [this install log]() may be helpful.
+
+### Windows Subsystem For Linux
+Installation on Windows Subsystem For Linux requires Windows Version 1809 (release due October 2018). 
+Tracked at #4346.
```

Comment:
Could you link to the right Repo here pls?

Reply:
done